CONTRIBUTING.md

```diff
@@ -9,94 +10,145 @@ it can be incorporated into the `main` branch.
-of MNE-BIDS are run through the BIDS validator to check if the conversion
-worked properly and produced datasets that conforms to the BIDS specifications.
+You should now have both the `mne` and `mne-bids` development versions available in your Python environment.
 
-You will need the `command line version` of the validator.
+### Install additional Python packages required for development
 
-#### Global (system-wide) installation
+Navigate to the root of the `mne-bids` repository and call:
 
-- First, install [Node.js](https://nodejs.org/en/).
-- For installing the **stable** version of `bids-validator`, please follow the
-instructions as detailed in the README of the bids-validator repository.
-- For installing the **development** version of `bids-validator`, see [here](https://github.com/bids-standard/bids-validator/blob/master/CONTRIBUTING.md#using-the-development-version-of-bids-validator).
+```Shell
+pip install -r test_requirements.txt
+pip install -r doc/requirements.txt
+```
 
-Test your installation by running:
+This will install several packages to run tests, and build the documentation for `mne-bids`.
 
-    $ bids-validator --version
+### Install the BIDS validator
 
-#### Local (per-user) development installation
+For a complete development setup of `mne-bids`, it is necessary to install the
+[BIDS validator](https://github.com/bids-standard/bids-validator).
+The outputs of `mne-bids` are run through the BIDS validator to check if the conversion
+worked properly and produced datasets conform to the
+[BIDS specification](https://bids-specification.readthedocs.io/en/stable/).
 
-Install [Node.js](https://nodejs.org/en/). If you're use `conda`, you can
-install the `nodejs` package from `conda-forge` by running
-`conda install -c conda-forge nodejs`.
+To install the **stable** version of `bids-validator`, please follow the instructions
+in the [bids-validator README](https://github.com/bids-standard/bids-validator#quickstart).
+You will need the "command line version" of the bids-validator.
 
-Then, retrieve the validator and install all its dependencies via `npm`.
+To install the **development** version of `bids-validator`, see the
+[contributor documentation of bids-validator](https://github.com/bids-standard/bids-validator/blob/master/CONTRIBUTING.md#using-the-development-version-of-bids-validator).
 
-    $ git clone git@github.com:bids-standard bids-validator.git
-    $ cd bids-validator/bids-validator
-    $ npm install
+We recommend that to get started, you install the **stable** `bids-validator`.
 
-Test your installation by running:
+Check your installation by running:
 
-    $ ./bin/bids-validator --version
+```Shell
+bids-validator --version
+```
 
+### Install GNU Make
 
-### Invoke pytest
+We use [GNU Make](https://www.gnu.org/software/make/) for developing `mne-bids`.
+We recommend that you install GNU Make and make use of our `Makefile` at the root
+of the repository.
```

Comment:
I don't think this is very helpful. We should instruct users how to get this "GNU Make" thing installed. The linked website looks like most GNU things: designed for people living in the 1980s.

Reply:
True, I update the instructions.